vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ElementSteps.java

```diff
@@ -197,7 +306,7 @@ public void doesEachElementByLocatorHaveChildWithLocator(Locator elementLocator,
      * @param cssName A name of the <b>CSS property</b>
      * @param cssValue An expected value of <b>CSS property</b>
     */
-    @Then("the context element has the CSS property '$cssName'='$cssValue'")
+    @Then("context element has CSS property `$cssName`=`$cssValue`")
```

Comment:
```suggestion
    @Then("context element has CSS property `$cssName` with value that $comparisonRule `$cssValue`")
```
where `$comparisonRule` is `StringComparisonRule`